.github/CODEOWNERS

```diff
@@ -11,14 +11,14 @@
 # All files except markdown are owned by dev team
 # -------------------------------------------------
 
-* @freecodecamp/dev-team
+* @freeCodeCamp/dev-team
```

Comment:
Please revert the changes to this file. GitHub's behaviour is a bit unpredictable with CODEOWNERS File

Reply:
> Please revert the changes to this file. GitHub's behaviour is a bit unpredictable with CODEOWNERS File

@raisedadead Sure.